lib/apis/v3/bgpconfig.go

```diff
@@ -67,6 +76,26 @@ type ServiceClusterIPBlock struct {
 	CIDR string `json:"cidr,omitempty" validate:"omitempty,net"`
 }
 
+// Community contains community value and its name.
+// BGP community value can be of format `aa:nn` or `aa:nn:mm`.
+// If `aa:nn` format is used, standard community will be used, where `aa` and `nn` are 16 bit number.
+// If `aa:nn:mm` format is used, large community will be used, where `aa`, `nn` and `mm` are 32 bit number.
+// Where, `aa` is an AS Number, `nn` and `mm` are per-AS identifier.
+type Community struct {
+	Name  string `json:"name,omitempty" validate:"required"`
+	Value string `json:"value,omitempty" validate:"required"`
+}
+
+// PrefixAdvertisement contains communities that should be applied to a route if it belong to a prefix.
+// `Communities` can be list of either community names already defined in `Specs.Communities` or community value of format `aa:nn` or `aa:nn:mm`.
+// If `aa:nn` format is used, standard community will be used, where `aa` and `nn` are 16 bit number.
```

Comment:
Same here - struct level comments should describe the struct, and field-level comments should describe the individual fields. 

This is important because these comments will get auto-generated into API documentation (eventually) and custom resource definitions.

Reply:
My suggestion for the comment here would be:

```
PrefixAdvertisement configures advertisement properties for the specified CIDR. 
```